taxonomy/emsi_client.py

```diff
@@ -153,6 +153,15 @@ def traverse_data(response):
         """
         Transform data to a more useful format.
         """
+        for skill_details in response['data']:
+            # append skill description in skill data extracted from "wikipediaExtract" tag
+            try:
+                desc = next(tag['value'] for tag in skill_details['skill']['tags'] if tag['key'] == 'wikipediaExtract')
+            except StopIteration:
+                LOGGER.warning('[TAXONOMY] "wikipediaExtract" key not found in skill: %s', skill_details['skill']['id'])
+                desc = ''
```

Comment:
I think we can simplify this whole block. Looks like next can take a default value which will be returned if we nothing is found. next docs https://docs.python.org/3/library/functions.html#next

Reply:
@muhammad-ammar Yes you are right, I did it this way so that I could log a warning as well.

Reply:
@HammadAhmadWaqas Is there a possibility that `wikipediaExtract` will not be present?

Reply:
`wikipediaExtract` is just a tag key, It is not mentioned in the docs it's a mandatory tag or not.
Added logs here to assure that if there will be missing description from our system, we will be able to confirm that is it due to the fact the API is not returning `WikipediaExtract`